src/components/App/App.js

```diff
@@ -20,11 +20,7 @@ const App = () => {
   return (
     <main>
       <Header isLoggedIn={isLoggedIn} setIsLoggedIn={setIsLoggedIn} />
```

Comment:
Hmmm, it looks like some changes that were made previously are going to get rewritten. We'll need to ensure that we don't use this version.